funnel/templates/project_updates.html.jinja2

```diff
@@ -6,21 +6,23 @@
 
 
 {% block base_content_inner %}
-  <div class="project-section" id="updates">
-    {% if project.features.post_update() -%}
-      <a class="update-form-block" href="{{ project.url_for('new_update') }}" aria-label="{% trans %}Add new update{% endtrans %}" data-cy-btn="add-update">
-        <div class="user">
-          <div class="user__box">
-            {{ useravatar(current_auth.user, add_profile_link=false) }}
-            <input type="text" placeholder="{% trans %}Post an update…{% endtrans %}" />
+  <div class="page-card page-card--nomargin">
+    <div class="project-section" id="updates">
+      {% if project.features.post_update() -%}
+        <a class="update-form-block" href="{{ project.url_for('new_update') }}" aria-label="{% trans %}Add new update{% endtrans %}" data-cy-btn="add-update">
+          <div class="user">
+            <div class="user__box">
+              {{ useravatar(current_auth.user, add_profile_link=false) }}
+              <input type="text" placeholder="{% trans %}Post an update…{% endtrans %}" />
+            </div>
           </div>
-        </div>
-      </a>
-    {%- endif %}
-    {{ updates_template() }}
-    {% if not project.published_updates -%}
-      <p class="mui-panel mui--bg-accent">{% trans %}This project has not posted any updates yet.{% endtrans %}</p>
-    {%- endif %}
+        </a>
+      {%- endif %}
+      {{ updates_template() }}
+      {% if not project.published_updates -%}
+        <p class="mui-panel mui--bg-accent">{% trans %}This project has not posted any updates yet.{% endtrans %}</p>
```

Comment:
Don't need `mui-panel` because this is already inside a card. Text can be simply "No updates", removing the word "posted" because the project is not an actor.